demo-app/src/open-truss/configs/8-mysql-demo.yaml

```diff
@@ -1,9 +1,21 @@
 workflow:
   version: 1
+  signals:
+    results: Records
   frames:
     - frame:
-      data:
-        query: select * from sys.metrics
-        source: mysql-demo
       view:
-        component: DataDemo
+        component: OTUqiDataProvider
```

Comment:
@jonmagic I updated the mysql demo to use the `OTUqiDataProvider` and everything just worked 🤘. I modified the kusto one too but didn't test it. I assume that should work as well.